libraries/AP_Camera/AP_Camera_Backend.cpp

```diff
@@ -241,6 +250,133 @@ void AP_Camera_Backend::send_camera_settings(mavlink_channel_t chan) const
         NaN);               // focusLevel float, percentage from 0 to 100, NaN if unknown
 }
 
+// calculate poi information
+void AP_Camera_Backend::calculate_poi()
+{
+#if AP_TERRAIN_AVAILABLE
+    if (!poi_calculation.sem.take_nonblocking()) {
+        return;
+    }
+    const AP_AHRS &ahrs = AP::ahrs();
+    if (!ahrs.get_location(poi_calculation.loc)) {
+        gcs().send_text(MAV_SEVERITY_ERROR, "POI: vehicle pos unavailable");
```

Comment:
How fast does this come out if you don'
t have a position available?
```suggestion
        GCS_SEND_TEXT(MAV_SEVERITY_ERROR, "POI: vehicle pos unavailable");
```
...also, ERROR is probably not a good level as it can happen from time-to-time.  INFO is probably reasonable if this message is required.